### PR TITLE
[PC-1762] Opta Digital Expansion Update

### PR DESCRIPTION
**Pull request by ArduinoBot**
## What This PR Changes
- Opta Digital Expansion Update - Inputs compatibility

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by jcarolinares** [comment]
@mcmchris please double check and try to fix the issues by yourself, if not ping me :) not urgent

**Comment by mcmchris** [comment]
This is updated with the current status of Main, conflicts resolved and ready to be merged

File: content/hardware/07.opta/opta-family/opta/tutorials/01.user-manual/content.md [hunk 1]
```diff
@@ -1719,6 +1723,16 @@ To learn more about Opta™ and the Arduino IoT Cloud, check out the following r
...
+| ----------------------- | ----------------------------------------------------------------------------------------------------- |
+| Supply Voltage          | 12...24 V                                                                                             |
+| Antipolarity protection | Yes                                                                                                   |
+| Overvoltage protection  | Yes (+20%)                                                                                            |
+| Inputs                  | 16x Digital (0-24 V) / Analog (0-10 V or 0-24 V) inputs                                                         |
+| Outputs                 | AFX00005: 8x Electromechanical Relays (250 VAC - 6 A), AFX00006: 8x Solid State Relays (24 VDC - 2 A) |
+| Degree of Protection    | IP20                                                                                                  |
+| Certifications          | FCC, CE, UKCA                                                                                         |
```
Comment (TaddyHC) [inline content/hardware/07.opta/opta-family/opta/tutorials/01.user-manual/content.md:1734]:
```suggestion
| Characteristics                     | Details                                                                                               |
|-------------------------------------|-------------------------------------------------------------------------------------------------------|
| Supply Voltage                      | 12...24 V                                                                                             |
| Antipolarity protection             | Yes                                                                                                   |
| Overvoltage protection              | Yes (+20%)                                                                                            |
| Maximum Supported Expansion Modules | Up to 5                                                                                               |
| Inputs                              | 16x Digital (0-24 V) / Analog (0-10 V or 0-24 V) inputs                                               |
| Outputs                             | AFX00005: 8x Electromechanical Relays (250 VAC - 6 A), AFX00006: 8x Solid State Relays (24 VDC - 3 A) |
| Degree of Protection                | IP20                                                                                                  |
| Certifications                      | FCC, CE, UKCA                                                                                         |
```

**Review by TaddyHC — APPROVED** [review]
Just one detail, LGTM!

**Comment by mcmchris** [comment]
Merging

**Pull request merged**